ocfweb/docs/docs/services/lab/lab-reservation-policy.md

```diff
@@ -0,0 +1,55 @@
+[[!meta title="Lab reservation policy"]]
+
+Registered Student Organizations of UC Berkeley are able to reserve the OCF lab
+space for a maximum of 2 hours at a time during the OCF’s regular operating
+hours. Those who reserve the OCF lab space are able to access the 28 computers
+within the lab (as well as additional seating), in addition to the main TV
+attached to the back wall. The OCF can additionally provide a whiteboard and
+printing (pursuant to printing quotas) to student groups who utilize the lab
+space. Should the lab be reserved, the option to fully close the lab to the
+outside population (excluding OCF front-desk staff) is readily available.
+
+Here are a few important things to note:
+
+- All OCF computers run on Debian Linux, so operating system specific
+  applications such as Microsoft Word will not be available. However,
+  browser-based tools, such as Google Docs, will work.
+
+- The OCF runs a decal from 8 to 9 pm on Tuesdays and Thursdays, and as such we
+  will be unable to allow lab reservations within that timeframe.
```

Comment:
What about BoD and staff meetings on Mondays?

Reply:
We may be willing to cancel Staff or BoD on any particular week (as we are doing for the FB event next week). Not saying we will say yes, but it is not unheard of.